### ISSUE TITLE
Generate python class and module scope docstrings

### ISSUE THREAD

**Issue by tekulvw** [kkoomen/vim-doge#657]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Class scope documentation serves to provide a description of what the class is and details on any class level parameters:

```py
class Foo:
    """
    Foo is a Thing that does Other Thing.
    :param baz: 
    """
    baz: int
```

Module scope documentation:

```py
"""
Foo serves to [...]
"""
import time

...
```

**Describe alternatives you've considered**

**Additional context**

**Comment by btalb** [comment]
+1.

Tracing to https://github.com/kkoomen/vim-doge/blob/master/helper/src/python/parser.rs and the associated tests it looks like only recognising functions is implemented.

It would be nice in the README to document what tags are supported for each language.

**Comment by kkoomen** [comment]
I've implement the PR #660 based on https://stackoverflow.com/a/3051356 with solely the addition of classes. I'm not supporting the import docstring because it's super rare to see that, but it's quite common to add class docs.

**Comment by kkoomen** [comment]
This feature has been merged and released in [v4.7.0](https://github.com/kkoomen/vim-doge/releases/tag/v4.7.0).

Feel free to submit any new issues if you experience any unwanted behavior in the future. Thanks for your contribution.
